src/main/java/org/interledger/Fulfillment.java

```diff
@@ -7,104 +7,152 @@
 
 /**
  * The fulfillment of a {@link Condition}.
```

Comment:
Gonna leave that off now and we can discuss further and address as part of #61.

I'm worried that talking about crypto-conditions will create more confusion given that we only use hashlocks but agree that we need better documentation on the subject.